src/test/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseDataTest.java

```diff
@@ -63,4 +66,23 @@ public void sortEvidenceByDate() {
         assertEquals("2019-02-01", sscsCaseData.getEvidence().getDocuments().get(1).getValue().getDateReceived());
         assertEquals("2019-01-01", sscsCaseData.getEvidence().getDocuments().get(2).getValue().getDateReceived());
     }
+
+    @Test
+    public void shouldCreateInfoRequest() throws JsonParseException, IOException {
```

Comment:
Not sure why the json needs to change. 🤷‍♂

Reply:
For InfoRequest class the json is creating with with its member infoRequests.appellantInfoRequest which causing a issue for case loader while updating a case which say validation errors=[ValidationResult(errorMessage=Field is not recognised, fieldId=infoRequests.appellantInfoRequest (not present in CCD definition)

Reply:
Code changed in order to generate correct Json infoRequests.appellantInfoRequestCollection